src/plugins/conditionals/conditionals.c

```diff
@@ -617,17 +616,15 @@ static CondResult parseConditionString (const Key * meta, const Key * suffixList
 	CondResult ret;
 	if ((ret = regcomp (&regex1, regexString1, REGEX_FLAGS_CONDITION)))
 	{
-		ELEKTRA_SET_OUT_OF_MEMORY_ERROR (parentKey,
-						 "Couldn't compile regex: most likely out of memory"); // the regex compiles so the only
+		ELEKTRA_SET_OUT_OF_MEMORY_ERROR (parentKey); // the regex compiles so the only
 		// possible error would be out of
 		// memory
 		ksDel (ks);
 		return ERROR;
 	}
 	if ((ret = regcomp (&regex2, regexString2, REGEX_FLAGS_CONDITION)))
 	{
-		ELEKTRA_SET_OUT_OF_MEMORY_ERROR (parentKey,
-						 "Couldn't compile regex: most likely out of memory"); // the regex compiles so the only
+		ELEKTRA_SET_OUT_OF_MEMORY_ERROR (parentKey); // the regex compiles so the only
```

Comment:
This is wrongly classified (at least at one place). `regerror()` tells what the error is. (see also above and below). Also remove the comment.